CardScan/Classes/ScanViewController.swift

```diff
@@ -304,6 +304,12 @@ import UIKit
         }
     }
     
+    public override func usePredictedCardNumber(predictedNumber: String?, currentFrameNumber: String) -> Bool {
```

Comment:
maybe change the name to `useCurrentFrameNumber` since I think that the `predictedNumber` here is the error corrected number

Reply:
you got it